pgx_show_hooks/Cargo.toml

```diff
@@ -12,7 +12,7 @@ pg15 = ["pgrx/pg15", "pgrx-tests/pg15"]
 pg_test = []
 
 [dependencies]
-pgrx = "0.8"
+pgrx = "~0.8.0"
```

Comment:
This seems to have the same meaning as `0.8`? Actually if there is an upgrade `Cargo.lock` will change to include the new version.

Reply:
I thought it was the same too, but if you see slack, it broke both my and Yuchen's builds.

Reply:
pgrx can both be a cargo-install target and a dependency. It looks fine to simply use either 0.8 or 0.8.0 here. Using 0.8.0 only matters when running cargo install?

Reply:
... that is to say, if you are referring to the cargo install thing, it does not seem to relate to the pgrx version here as a dependency.